### ISSUE TITLE
[AvatarStack]: accessibility issues related to the component

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1458]
### Description
Reading order with screen reader doesn't match visual order

### Precondition
Screen Reader is **ON**

### Steps to Reproduce
1. Open [AvatarStack](https://uui.epam.com/documents?category=components&id=avatarStack&mode=doc&skin=UUI4_promo) component.
2. Use the `Down` arrow key to navigate to the image avatars and observe.

### Actual Result
Reading order with screen reader doesn't match visual order: starts from the last avatar and ends with the first one.

https://github.com/epam/UUI/assets/57751283/de35811e-cccf-4431-9d7a-a3cd829617ee

### Expected Result
Reading order with screen reader should match visual order. 
**Failed WCAG Checkpoint:** 1.3.2 Meaningful Sequence (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

**Comment by NatalliaAlieva** [comment]
Not fixed

https://github.com/epam/UUI/assets/57751283/75a24f4f-2620-4ab3-8c93-cc98bd38f0b9


https://github.com/epam/UUI/assets/57751283/0258e749-9030-439f-9ff2-2f848114811a

**Comment by NatalliaAlieva** [comment]
This ticket moved in [this](https://github.com/epam/UUI/issues/1553) bug, case №44.
